core/src/trezor/ui/model/button/t1.py

```diff
@@ -0,0 +1,175 @@
+from trezor import ui
+from trezor.ui import display, in_area
+
+from ..types import ButtonStyle, ButtonStyleState
+
+if False:
+    from typing import Optional
+    from ..types import (  # noqa: F401
+        ButtonStyleStateType,
+        ButtonStyleType,
+        ButtonContent,
+    )
+
+
+class ButtonDefault(ButtonStyle):
+    class normal(ButtonStyleState):
+        bg_color = ui.FG
+        fg_color = ui.BG
+        text_style = ui.BOLD
+        border_color = ui.FG
+        radius = 1
+
+    class active(normal):
+        bg_color = ui.BG
+        fg_color = ui.FG
+        text_style = ui.BOLD
+        border_color = ui.BG
+        radius = 1
+
+    class disabled(normal):
+        bg_color = ui.FG
+        fg_color = ui.BG
+        text_style = ui.NORMAL
+        border_color = ui.FG
+        radius = 1
+
+
+class ButtonMono(ButtonDefault):
+    class normal(ButtonDefault.normal):
+        text_style = ui.MONO
+
+    class active(ButtonDefault.active):
+        text_style = ui.MONO
+
+    class disabled(ButtonDefault.disabled):
+        text_style = ui.MONO
+
+
+class ButtonMonoDark(ButtonDefault):
+    class normal(ButtonDefault.normal):
+        bg_color = ui.BG
+        fg_color = ui.FG
+        text_style = ui.MONO
+        border_color = ui.BG
+        radius = 0
+
+    class active(normal):
+        bg_color = ui.FG
+        fg_color = ui.BG
+        text_style = ui.MONO
+        border_color = ui.FG
+        radius = 0
+
+    class disabled(normal):
+        bg_color = ui.BG
+        fg_color = ui.FG
+        text_style = ui.MONO
+        border_color = ui.BG
+        radius = 0
+
+
+class ButtonConfirm(ButtonDefault):
+    pass
+
+
+class ButtonCancel(ButtonDefault):
+    class normal(ButtonDefault.normal):
+        bg_color = ui.BG
+        fg_color = ui.FG
+        border_color = ui.BG
+        radius = 0
+
+    class active(ButtonDefault.active):
+        bg_color = ui.FG
+        fg_color = ui.BG
+        border_color = ui.FG
+        radius = 0
+
+    class disabled(ButtonDefault.disabled):
+        bg_color = ui.BG
+        fg_color = ui.FG
+        border_color = ui.BG
+        radius = 0
+
+
+class ButtonAbort(ButtonDefault):
+    pass
+
+
+class ButtonClear(ButtonCancel):
+    pass
+
+
+class ButtonMonoConfirm(ButtonDefault):
+    class normal(ButtonDefault.normal):
+        text_style = ui.MONO
+
+    class active(ButtonDefault.active):
+        text_style = ui.MONO
+
+    class disabled(ButtonDefault.disabled):
+        text_style = ui.MONO
+
+
+def render_button(
+    text: Optional[str], icon: Optional[bytes], s: ButtonStyleStateType, area: ui.Area
+) -> None:
+    if in_area(area, 0, ui.HEIGHT - 1):
+        is_right = False
+    elif in_area(area, ui.WIDTH - 1, ui.HEIGHT - 1):
+        is_right = True
+    else:
+        raise AssertionError
+
+    ax, ay, aw, ah = area
```

Comment:
this doesn't do anything istm?

Reply:
Fixed 2ccf3a6fa3958e065e0255371f53a4f86887e0ec. I'd expect flake8 to catch this:thinking:.